Lib/inspect.py

```diff
@@ -1638,7 +1638,29 @@ def getclosurevars(func):
 
 # -------------------------------------------------- stack frame extraction
 
-Traceback = namedtuple('Traceback', 'filename lineno function code_context index')
+_Traceback = namedtuple('_Traceback', 'filename lineno function code_context index')
+
+class Traceback(_Traceback):
```

Comment:
Let's add `__slots__` in case people make a lot of these things.
```suggestion
class Traceback(_Traceback):
    __slots__ = ('positions',)
```

Reply:
You cannot do `__slots__` if you inherit from a named tuple.